### DISCUSSION TITLE
关于 Badge 组件开发相关

### DISCUSSION THREAD

**Discussion by xluoyu** [FightingDesign/fighting-design#28] [Ideas]
## 功能
显示数字、状态等信息的角标

## 参数
| 参数 | 类型 | 说明 |
| - | - | - |
| `value` | `String` \| `Number` | 显示值 |
| `max` | `Number` | `value`为数字时的最大值，超出显示为`${value}+` |
| `isDot` | `Boolean` |  是否展示为圆点 |
| `hidden` | `Boolean` |  是否显示 |
| `type` |  `'primary' \| 'success' \| 'danger' \| 'warning' ` | 类型(颜色区分) |

## slots
* `default`  默认内容

**Comment by Tyh2001** [comment]
写的不错~
